bukkit/src/main/java/net/william278/huskhomes/hook/RedisEconomyHook.java

```diff
@@ -0,0 +1,26 @@
+package net.william278.huskhomes.hook;
+
+import dev.unnm3d.rediseconomy.api.RedisEconomyAPI;
+import net.william278.huskhomes.HuskHomes;
+import net.william278.huskhomes.HuskHomesInitializationException;
+import org.jetbrains.annotations.NotNull;
+
+public class RedisEconomyHook extends VaultEconomyHook {
+
+
+    public RedisEconomyHook(@NotNull HuskHomes implementor) {
+        super(implementor);
+    }
+
```

Comment:
Could we do it so there's only one space between class header and the constructor and the constructor and initialize method and instead of two?

Reply:
grrrrr